problem-3/problem-3.mojo

```diff
@@ -1,25 +1,23 @@
-from python import Python
+import math
 
-def makeGood(s: String):
-    py = Python.import_module("builtins")
-    result = str("")
+fn makeGood(s: String):
+    var result = str("")
 
-    i = 0
+    var i = 0
     while i < len(s):
-        
-        if i < len(s) - 1 and py.abs(ord(s[i]) - ord(s[i + 1])) == 32:
```

Comment:
I could see why importing from python was not necessary in my other codes but here this should be working just fine, no?